pkg/cmd/pipeline/start.go

```diff
@@ -385,6 +394,19 @@ func (opt *startOptions) startPipeline(pName string) error {
 		},
 	}
 
+	if opt.DryRun != "" {
+		prBytes, err := yaml.Marshal(pr)
+		if err != nil {
+			return err
+		}
+
+		if err := ioutil.WriteFile(opt.DryRun, prBytes, os.ModePerm); err != nil {
```

Comment:
I do not have unit tests for this since it writes a local file. Would it make sense to write the file and then delete it as part of the unit test?

Reply:
not sure if that's the proper way but that's how i would do it 😅

Reply:
Sure, I'll try it.

Reply:
Wondering if this approach is the right way here, but there are at least unit tests for errors/marshaling the pipelinerun.